src/src/WebServer/Markup_Forms.cpp

```diff
@@ -294,6 +294,27 @@ void addFormIPBox(const String& label, const String& id, const uint8_t ip[4])
   addHtml('>');
 }
 
+// ********************************************************************************
+// Add a MAC Box form
+// ********************************************************************************
+void addFormMACBox(const String& label, const String& id, const MAC_address mac)
+{
+  addRowLabel_tr_id(label, id);
+
+  String html;
+  html.reserve(80 + id.length());
```

Comment:
This could use `addHtml()`, instead of first assembling it into a String?

Reply:
Check!